datadog/resource_datadog_monitor.go

```diff
@@ -418,6 +431,17 @@ func resourceDatadogMonitorRead(d *schema.ResourceData, meta interface{}) error
 		d.Set("query_config", queryConfig)
 	}
 
+	// Update the state with the contents of the API + whats in the config
```

Comment:
I think there's one problem in this situation:

* create a downtime for some scope that’s a specific timestamp in the future
* unmute that scope in the ui
* run `terraform apply`
=> no change in plan even though the resource in the config is clearly different from current state